.buildkite/pipeline.yml

```diff
@@ -1,16 +1,20 @@
 steps:
-  - label: 'stack coveralls coverage'
-    command:
-      - 'nix-shell -A runCoveralls'
```

Comment:
do the new jobs still run coveralls?

Reply:
No, I thought I'd removed that :-/

Reply:
I'll add it back in.

Reply:
I added it back in and later commented it out due to errors in the build that I"m not convinced are Nix related.